impeller/playground/playground.cc

```diff
@@ -334,7 +334,8 @@ std::optional<DecompressedImage> Playground::LoadFixtureImageRGBA(
 }
 
 std::shared_ptr<Texture> Playground::CreateTextureForFixture(
-    const char* fixture_name) const {
+    const char* fixture_name,
+    std::optional<size_t> mip_count) const {
```

Comment:
Can this just be `enable_mipmapping`? Specifying the mip counts explicitly is perhaps a too expressive for what we need in the playgrounds.

Reply:
Done.